packages/clerk-js/tsconfig.json

```diff
@@ -20,8 +20,10 @@
     "useUnknownInCatchVariables": false,
     "declaration": false,
     "jsx": "react-jsx",
-    "jsxImportSource": "@emotion/react"
-    // "noUncheckedIndexedAccess": true // TODO: Enable and progressively fix related errors
```

Comment:
@nikosdouvlis Shall we add a ticket to bring `noUncheckedIndexedAccess` back ?

Reply:
This wasn't changed as part of this PR but agreed, let's add a ticket in triage